.github/workflows/release.yml

```diff
@@ -90,16 +93,27 @@ jobs:
           asset_name: ${{ format('vscode-codeql-{0}.vsix', steps.prepare-artifacts.outputs.ref_name) }}
           asset_content_type: application/zip
 
+
+      - name: No Release
+        if: github.event_name != 'pull_request'
+        run: |
+          echo "Not making a release because this is not a pull request"
```

Comment:
Why are all these changes here?

Reply:
Darn...must be something about the merging I've been doing to keep it up to date with `main`. This should already be in main, but I'll check to make sure.

Reply:
I think it may be worth rebasing this branch for a clean diff, rather than merging in main. Your changes to the release workflow on main are more recent.

Reply:
Actually, these changes were performed as part of the telemetry change in order to handle the APPLICATION_INSIGHTS key. It's not directly related to telemetry, but I cleaned up the yml as part of it.

It's a bit of a messy rebase and one that I've half performed several times already. Probably in the end, I should just squash this all into a single commit.